src/client/components/app/Navigation/NavSidebar.tsx

```diff
@@ -85,19 +87,20 @@ const RouterLink = styled(Link)<{ selected: boolean }>`
 const StyledSidebar = styled.div<{ collapsed?: boolean }>`
   display: flex;
   flex-direction: column;
-  color: ${({ theme }) => theme.colors.primaryVariant};
-  background-color: ${({ theme }) => theme.colors.primary};
+  color: ${({ theme }) => theme.colors.icons.variant};
+  background-color: transparent;
```

Comment:
Is this needed?

Reply:
Nope, just a vestige from testing. We should add this things on a ticket to fix later